documentation/tutorial.md

```diff
@@ -68,3 +71,622 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+    wMajorityWriteAvailabilityDate: ISODate("2022-12-02T11:24:09.651Z")
+  },
+  members: [
+    {
+      _id: 0,
+      name: '10.23.62.156:27017',
+      health: 1,
+      state: 1,
+      stateStr: 'PRIMARY',
+      uptime: 11747,
+      optime: { ts: Timestamp({ t: 1669991990, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-12-02T14:39:50.000Z"),
+      lastAppliedWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastDurableWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      syncSourceHost: '',
+      syncSourceId: -1,
+      infoMessage: '',
+      electionTime: Timestamp({ t: 1669980249, i: 2 }),
+      electionDate: ISODate("2022-12-02T11:24:09.000Z"),
+      configVersion: 5,
+      configTerm: 1,
+      self: true,
+      lastHeartbeatMessage: ''
+    },
+    {
+      _id: 1,
+      name: '10.23.62.55:27017',
+      health: 1,
+      state: 2,
+      stateStr: 'SECONDARY',
+      uptime: 305,
+      optime: { ts: Timestamp({ t: 1669991990, i: 1 }), t: Long("1") },
+      optimeDurable: { ts: Timestamp({ t: 1669991990, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-12-02T14:39:50.000Z"),
+      optimeDurableDate: ISODate("2022-12-02T14:39:50.000Z"),
+      lastAppliedWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastDurableWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastHeartbeat: ISODate("2022-12-02T14:39:51.868Z"),
+      lastHeartbeatRecv: ISODate("2022-12-02T14:39:51.882Z"),
+      pingMs: Long("0"),
+      lastHeartbeatMessage: '',
+      syncSourceHost: '10.23.62.156:27017',
+      syncSourceId: 0,
+      infoMessage: '',
+      configVersion: 5,
+      configTerm: 1
+    },
+    {
+      _id: 2,
+      name: '10.23.62.243:27017',
+      health: 1,
+      state: 2,
+      stateStr: 'SECONDARY',
+      uptime: 300,
+      optime: { ts: Timestamp({ t: 1669991990, i: 1 }), t: Long("1") },
+      optimeDurable: { ts: Timestamp({ t: 1669991990, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-12-02T14:39:50.000Z"),
+      optimeDurableDate: ISODate("2022-12-02T14:39:50.000Z"),
+      lastAppliedWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastDurableWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastHeartbeat: ISODate("2022-12-02T14:39:51.861Z"),
+      lastHeartbeatRecv: ISODate("2022-12-02T14:39:52.372Z"),
+      pingMs: Long("0"),
+      lastHeartbeatMessage: '',
+      syncSourceHost: '10.23.62.55:27017',
+      syncSourceId: 1,
+      infoMessage: '',
+      configVersion: 5,
+      configTerm: 1
+    }
+  ],
+  ok: 1,
+  '$clusterTime': {
+    clusterTime: Timestamp({ t: 1669991990, i: 1 }),
+    signature: {
+      hash: Binary(Buffer.from("dbe96e73cf659617bb88b6ad11152551c0dd9c8d", "hex"), 0),
+      keyId: Long("7172510554420936709")
+    }
+  },
+  operationTime: Timestamp({ t: 1669991990, i: 1 })
+}
+```
+
+Now exit the shell by typing:
+```
+exit
+```
+
+
+### Remove replicas:
+To remove a replica type:
+```
+juju remove-unit mongodb/2
```

Comment:
It is probably worth expanding on this a bit here. It is obvious to you how the unit numbers work etc, but not to people just blindly pasting things in. Perhaps expand this by demonstrating how you can list the current replicas first, before choosing which unit to remove?